### PR TITLE
Update introductory Apache Airflow pipeline tutorial for Elyra v3 

### PR DESCRIPTION
**Pull request by ptitzler**
This PR replaces the existing "Hello world Apache Airflow" tutorial. The tutorial focuses on generic pipelines. A separate tutorial will be provided in the future that covers runtime-specific pipelines.

- [X] The new artifacts include a pipeline file that needs to be migrated to the v4 metadata version.
- [X] The README contains one of more references to documentation topics that will only properly resolve after version 3.0 was published.
- [x] Upgraded to pipeline v4
 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: pipelines/run-generic-pipelines-on-apache-airflow/README.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+   Notebooks and scripts can be parameterized using environment variables. The node you are looking at requires a variable that identifies the download location of a data file.
+
+   ![Declare environment variables](doc/images/environment-variable-declarations.png)
+
+   Refer to [_Best practices for file-based pipeline nodes_](https://elyra.readthedocs.io/en/stable/user_guide/best-practices-file-based-nodes.html) in the _User Guide_ to learn more about considerations for each configuration setting.
+
+### Define a runtime environment configuration
+
+A [runtime configuration](https://elyra.readthedocs.io/en/stable/user_guide/runtime-conf.html) in Elyra contains connectivity information for an Apache Airflow instance and S3-compatible cloud storage. In this tutorial you will use the GUI to define theconfiguration, but you can also use the CLI.
+
```
Comment (kiersten-stokes) [inline pipelines/run-generic-pipelines-on-apache-airflow/README.md:120]: Missing the space between `the` and `configuration`

File: pipelines/run-generic-pipelines-on-apache-airflow/README.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+The DAG run outputs (completed notebooks, script output, and declared output files) are persisted in the cloud storage bucket you've configured in the runtime configuration.
+
+Elyra does not automatically download the output artifacts to your JupyterLab environment from the cloud storage bucket after DAG execution has completed. Therefore you have to use an S3 client, or, if configured, the cloud storage's web GUI to access the artifacts.
+
+1. Navigate to the bucket you've specified in the runtime configuration to review the content. 
+
+   ![Review run output artifacts](doc/images/inspect-object-storage-bucket-content.png)
+
+   The bucket contains for each node the following artifacts, which are prefixed with the DAG name:
+      - a `tar.gz` archive containing the notebook or script, and, if applicable, its declared input file dependencies
```
Comment (kiersten-stokes) [inline pipelines/run-generic-pipelines-on-apache-airflow/README.md:222]: Same as the Kubeflow tutorial where some commas around `for each node` may help readability

**Review by kiersten-stokes — COMMENTED** [review]
Some NITs!

**Comment by ptitzler** [comment]
> Some NITs!

Thank you! Fixed!

**Review by kiersten-stokes — APPROVED** [review]
LGTM!

**Review by kiersten-stokes — APPROVED** [review]
LGTM!

**Review by kiersten-stokes — APPROVED** [review]

**Pull request merged**